app/src/main/scala/org/alephium/explorer/service/TokenService.scala

```diff
@@ -267,7 +268,7 @@ object TokenService extends TokenService with StrictLogging {
   ): Future[Unit] = {
     listTokenWithoutInterfaceId()
       .flatMap { tokens =>
-        Future.sequence(tokens.map(token => fetchAndStoreTokenMetadata(token, blockFlowClient)))
+        foldFutures(tokens) { token => fetchAndStoreTokenMetadata(token, blockFlowClient) }
```

Comment:
I had this issue before, but couldn't find the issue. Now I know that we can't update too much tokens at the same time. For example here with the `Future.sequence` it updates all ANS nft at the same time and the DB doesn't like it.

Safer to update sequentially.